gangplank/ocp/bc.go

```diff
@@ -271,7 +261,7 @@ binary build interface.`)
 			case <-ctx.Done():
 				break
 			case <-terminate:
-				return nil
+				break
```

Comment:
I'm trying to understand what's going on here.  What does the break statement do here functionally?  If we catch the interrupt signal and put true in the terminate channel, it gets read from the channel here and breaks out of the the select but we're still in the outer loop.  Does the cancel() call prevent pods from being created?

Reply:
Indeed, you are correct. I've cleaned it up and push a better fix.